ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+### Context
+
+* [ ] **mapbox-gl-js version**: 
+* [ ] **browser version**: 
+* [ ] **operating system**: 
```

Comment:
We don't need checkboxes for this list. The completeness of each item is determined by whether or not the field is "filled in".